boxcli/featureflag/feature.go

```diff
@@ -0,0 +1,34 @@
+package featureflag
+
+import (
+	"os"
+)
+
+type feature struct {
+	name    string
+	enabled bool
+}
+type Feature interface {
```

Comment:
Why an interface?

Reply:
You are right, this is overkill, I don't imagine us passing feature flag objects outside this package. Will rm